### ISSUE TITLE
Solution might be too wordy 

### ISSUE THREAD

**Issue by kiwibang** [kiwibang/pe#16]
![Screenshot 2023-11-17 at 5.29.12 PM.png](https://raw.githubusercontent.com/kiwibang/pe/main/files/0deae2ff-dc4b-45c5-946d-96ec16579773.png)

Maybe the steps can be broken up more?

**Comment by nus-se-script** [comment]
# Team's Response
We went with a detailed solution because manually editing `.json` files is a very risky operation. There is also a disclaimer that says that only users that know the structure of `.json` files should read this, therefore most users would skip this section of the guide.

If there are users that need this section of the guide, They already have the patience to manually edit .json files, which is more harder than reading this part of the guide.
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.NotInScope`]

- [x] I disagree

**Reason for disagreement:** The user might be patient enough to edit .json files, but might not be patient enough to read this section. 

This issue that I raise falls under several categories:

- The explanation is too brief or unnecessarily long.
- hard to read/understand
- The visuals are not well integrated to the explanation
- looks messy

I chose the last category as the main issue. Definitely not not in scope

<catcher-end-of-segment><hr>
